ui/css/base-styles.scss

```diff
@@ -17,6 +17,30 @@ body {
 
 html {
   min-height: 500px;
+  background-color: var(--color-background-alternative);
+
+  @include screen-sm-max {
+    &:not([data-theme]) {
+      background-color: var(--color-background-default);
+    }
+  }
+
+  /*
+    Until we get the user's preference from React state,
+    assume the user is in dark mode, because the default
+    theme is system preference
+  */
+  @media (prefers-color-scheme: dark) {
+    &:not([data-theme]) {
+      background-color: var(--brand-colors-grey-grey900);
```

Comment:
We may have to include color here too to fix the errors page
```suggestion
      color: var(--brand-colors-white-white000);
      background-color: var(--brand-colors-grey-grey900);
```

<img width="361" alt="Screenshot 2023-08-16 at 9 36 43 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/cf9f067b-f2c5-497a-a319-3d915fc22527">

Reply:
I think we are missing something in this solution, I am still seeing this issue here.
![Screenshot 2023-08-18 at 9 37 51 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/eacbf036-c218-42a0-bb57-4d406980ee75)

Reply:
Done!